### ISSUE TITLE
Authentication failure on cluster connect

### ISSUE THREAD

**Issue by idar** [EventStore/EventStore#2721]
I am trying to install 20.6 on ubuntu 20.04.
I have no gotten it to start and not complain about certificates anymore.

Configfile:
`---
RunProjections: All
StartStandardProjections: True
ClusterSize: 3
ClusterDns: tl-eventstore-svarut-test.i04.local
IntIp: 10.241.11.252
ExtIp: 10.241.11.252
TrustedRootCertificatesPath: /etc/eventstore/ca
CertificateFile: apfx file
CertificatePassword: apassword
`
When it start i get this error:
[98436,20,15:19:50.986,WRN] Failed authorization check for "(anonymous)" in 00:00:00.0000045 with "node/gossip : update Deny : Policy : Legacy 1 12/31/9999 23:59:59 +00:00 : default:denied by default:Deny, $"

I am a bit stuck since 20.06 have almost no documentation.

**Comment by alexeyzimarev** [comment]
It is because the common name for certificates doesn't match what the cluster node expects.

It is mentioned in the [release notes](https://eventstore.com/blog/event-store-20.6.0-release/):
> The certificate’s subject needs to be CN=eventstoredb-node.

The easiest way to do it right is to use the [tool we created](https://github.com/EventStore/es-gencert-cli) to generate proper certificates.

I'd also rather recommend using 20.6.1 where a few common issues have been resolved.
